src/lib/core/CHIPTLVReader.cpp

```diff
@@ -1573,5 +1573,36 @@ CHIP_ERROR TLVReader::GetNextPacketBuffer(TLVReader & reader, uintptr_t & bufHan
     return CHIP_NO_ERROR;
 }
 
+/**
+ * Get the destination reader with the given tag
```

Comment:
Maybe:

> Position the destination reader on the next element with the given tag within its current containment context?

or so for the general function description?  Is that the intended behavior?  If there are multiple elements with the given tag value it'll pick the first one; that should be clearly documented, at least.

Reply:
Thanks for your suggestion! just update it.